src/app/shared/enum/enumUA/provider-admin.ts

```diff
@@ -14,4 +14,16 @@ export enum ProviderAdminTitles {
   NeverLogged = 'Очікує логування',
   Accepted = 'Підтвердженно',
   Blocked = 'Заблоковано'
+}
+
+export enum providerUserRole {
```

Comment:
specify the enum that it is for tab naming
and I would like to merge it to the same enum because you cooypaste Каьінет Надавача
export enum PersonalCabinetTitle{
provider = 'КАБІНЕТ НАДАВАЧА',
  admin = 'КАБІНЕТ ТЕХ. АДМІНА',
  user = 'КАБІНЕТ КОРИСТУВАЧА',
 deputy = 'КАБІНЕТ ЗАСТУПНИКА ДИРЕКТОРА',
  ProviderAdmin = 'КАБІНЕТ АДМІНІСТРАТОРА ГУРТКА',
}